.github/workflows/build-devcontainer.yaml

```diff
@@ -66,8 +60,9 @@ jobs:
           push:
             ${{ github.ref == 'refs/heads/main' || startsWith(github.ref,
             'refs/tags/') }}
+          # As above re `prql`
           cache-from: |
-            ghcr.io/${{ github.repository_owner }}/prql-devcontainer-base
+            ghcr.io/prql/prql-devcontainer-base
```

Comment:
Maybe `docker/metadata-action`'s output can be used here?

https://github.com/docker/metadata-action#outputs